_data/Mubarraqqq/README.md

```diff
@@ -0,0 +1,94 @@
+# [Mubaraq](https://github.com/Mubarraqqq/)'s Zarr Collection
+
+
+## Zarr Use Case
+- Demonstration of how we may use Zarr to support streaming data from simulation models to analysis environment - [Click here](https://medium.com/pangeo/streaming-zarr-ccf0d518b1c0)
+- Preparing versions of NWM Reanalysis dataset in Zarr format - [Click here](https://github.com/azavea/noaa-hydro-data/blob/master/src/esip-2022-presentation/save_zarr_data.ipynb)
+- Using Zarr to access cloud storage just like your local file system - [Click here](https://measurespace.medium.com/use-zarr-to-access-clound-storage-just-like-your-local-file-system-d67607cb128b)
+
+
+## Tweet Mentions
+
+- https://twitter.com/rajadain/status/1574097641469841410
+- https://twitter.com/notjustmoore/status/1579536653852766208
+- https://twitter.com/azavea/status/1573036727169871872
+- https://twitter.com/notjustmoore/status/1256232842755014656
+
+
+## Zarr Data Visualisation
+
+#### 30GB [microscopic image](https://twitter.com/notjustmoore/status/1256232842755014656) of covid19 newly published and converted to zarr stored in Embassy S3 and viewed with an [openmicroscopy plugin](https://www.openmicroscopy.org/bio-formats/) :
+
+![image1](/_data/Mubarraqqq/images/Microscopy1.jpg)
```

Comment:
The visualisation that you've added here and all of the visualisations down below are not rendering. Can you please correct it? Thanks!

Reply:
Alright. I will do that right away!

Reply:
Hello **Mr. Sanket**, I have made the required changes to render my visualization.